<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DRuGLORD" failures="9" tests="16" name="com.cab.cucumber.runners.RunnerUsingTestNG" time="56.664" errors="0" timestamp="2025-08-04T10:04:17 IST" skipped="0">
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.048">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.326">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="7.012"/>
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.114">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="2.235"/>
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="7.631"/>
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.202">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="2.550"/>
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.068">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="2.367"/>
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.120">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="2.974">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="4.464">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.123"/>
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="3.197">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.cab.cucumber.definitions.Booking.verify_the_invalid_data(Booking.java:80)
at ✽.verify the invalid data(file:///C:/Users/hp/eclipse-workspace/AgileQADelivery-TTP1QESA2520-Group5/src/test/java/com/cab/cucumber/features/Booking.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.cab.cucumber.runners.RunnerUsingTestNG" name="runScenario" time="2.233"/>
</testsuite> <!-- com.cab.cucumber.runners.RunnerUsingTestNG -->
